external/avro/src/test/scala/org/apache/spark/sql/execution/benchmark/AvroWriteBenchmark.scala

```diff
@@ -19,22 +19,17 @@ package org.apache.spark.sql.execution.benchmark
 
 /**
  * Benchmark to measure Avro data sources write performance.
- * Usage:
- * 1. with spark-submit: bin/spark-submit --class <this class> <spark sql test jar>
- * 2. with sbt: build/sbt "avro/test:runMain <this class>"
+ * To run this benchmark:
+ * {{{
+ *   1. without sbt: bin/spark-submit --class <this class>
```

Comment:
Please add `avro`:
```
bin/spark-submit --class <this class> --jars ./core/target/spark-core_2.11-3.0.0-SNAPSHOT-<spark core test jar>,<spark catalyst test jar>,<spark sql test jar> <spark avro test jar>
```

Reply:
@yucai Good catch! I think it needs `<spark avro jar>`, added.